feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/quicksettings/ui/QuickSettingsComponents.kt

```diff
@@ -220,23 +220,23 @@ fun QuickFlipCamera(
 
 @Composable
 fun QuickSetCaptureMode(
-    setCaptureMode: (CaptureMode) -> Unit,
-    currentCaptureMode: CaptureMode,
+    setCaptureMode: (StreamConfig) -> Unit,
+    currentStreamConfig: StreamConfig,
     modifier: Modifier = Modifier,
     enabled: Boolean = true
 ) {
     val enum: CameraCaptureMode =
```

Comment:
It seems like maybe the `CameraCaptureMode` enum should also be renamed to `CameraStreamConfig`?